va/va.go

```diff
@@ -80,20 +94,37 @@ func NewValidationAuthorityImpl(
 		},
 		[]string{"type", "result"})
 	stats.MustRegister(validationTime)
+	remoteValidationTime := prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name: "remote_validation_time",
+			Help: "Time taken to remotely validate a challenge",
+		},
+		[]string{"type"})
```

Comment:
So this is kind of weird, `remote_validation_time` is meant to be the total time for all remote validations to complete, the comment could probably be better worded. The individual remote hosts will report their own `validation_time` if we scrap them for metrics (which we definitely should) with their own `instance` field which individually identifies them.

If we don't want to scrape the remote instances (which I personally think is a bad idea) we could add a different metric or change this metric to be per remote and in that case we'd want to add a special `remote_name` or `addr` or something label.